src/livestreamer/plugins/media_ccc_de.py

```diff
@@ -0,0 +1,220 @@
+    """Filter all availabe live streams for given json and room name.
+
+    API-Doku: https://github.com/voc/streaming-website#json-api
+
+    :param json_string: json as string
+    :param room_from_url:
+
+    """
+    json_object = create_json_object(json_string)
+
+    streams = {}
+    for group in json_object:
+        for room in group['rooms']:
+            # only consider to requested room
+            match = _url_streaming_media_re.match(room['link'])
+            if not match.group('room') == room_from_url:
+                continue
+
+            for stream in room['streams']:
+                # get stream language
+                if stream['isTranslated'] == False:
+                    language = 'native'
+                else:
+                    language = 'translated'
+
+                # get available hls stream urls
+                if 'hls' in stream['urls'].keys():
+                    stream_url = stream['urls']['hls']['url']
+                    name = None
+                    # native HLS streams are announced as
+                    # ${height}p and (hd|sd)_native_${height}p
+                    if language == 'native':
+                        name          = "%sp" % stream['videoSize'][-1]
+                        long_name     = "hls_%s_%sp" % ("native",\
+                                                        stream['videoSize'][-1])
+                        streams[name]      = stream_url
+                        streams[long_name] = stream_url
+                    elif language == 'translated':
+                        long_name     = "hls_%s_%sp" % ("translated",\
+                                                        stream['videoSize'][-1])
+                        streams[long_name] = stream_url
+
+                # get available audio only mpeg urls
+                if 'mp3' in stream['urls'].keys():
+                    stream_url    = stream['urls']['mp3']['url']
+                    name          = "audio_%s_mpeg" % language
+                    streams[name] = stream_url
+
+                # get available audio only opus urls
+                if 'opus' in stream['urls'].keys():
+                    stream_url    = stream['urls']['opus']['url']
+                    name          = "audio_%s_opus" % language
+                    streams[name] = stream_url
+
+    return streams
+
+
+class media_ccc_de(Plugin):
+    @classmethod
+    def can_handle_url(self, url):
+        return _url_media_re.search(url) or _url_streaming_media_re.search(url)
+
+    def _get_streams(self):
+        streams = {}
+
+        # streaming.media.ccc.de
+        match = _url_streaming_media_re.match(self.url)
+        if match:
+            query_url    = API_URL_STREAMING_MEDIA
+            live_streams = parse_streaming_media_json(get_page(query_url),\
+                                                      match.group('room'))
+
+            for stream_name in live_streams.keys():
```

Comment:
You can do `for stream_name, stream_url in live_streams.items()` to get both the key and the value.